src/App.tsx

```diff
@@ -1,15 +1,13 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
-// tslint:disable: ordered-imports
```

Comment:
Why are you removing this?

Reply:
why not? i think we should avoid tslint disable comments in our code so just order the imports instead

Reply:
TSLint is supposed to increase readability and enforce standards. I purposefully put it there so the order of the imports will match the order of the Routes, which makes it easier to pinpoint imports. Alphabetical ordering doesn't help here.